core/files/css/base.php

```diff
@@ -696,6 +696,22 @@ private function get_style_control_value( array $control, array $values ) {
 
 		$value = $values[ $control['name'] ];
 
+		// If the control value is empty, and the control has a global default set, fetch the global value and use it.
```

Comment:
Why not to move this code to another method?

Reply:
How would you do it? @KingYes